dags/fxa_events.py

```diff
@@ -106,6 +106,7 @@
 
     fxa_users_services_first_seen = bigquery_etl_query(
         task_id='fxa_users_services_first_seen',
+        project_id='moz-fx-data-shared-prod',
         sql_file_path='sql/telemetry_derived/fxa_users_services_first_seen_v1/init.sql',
```

Comment:
Do we want to replace `sql_file_path` here too?

```suggestion
        destination_table='fxa_users_services_first_seen_v1',
```

Reply:
No, this is a weird case. We recreate this table in its entirety every day rather writing just a new partition. We have to specify the full path to `init.sql` even with this change.